user/for-beginners.md

```diff
@@ -1,57 +1,53 @@
+title: Travis CI for Beginners
 layout: en
 permalink: /user/for-beginners/
 ---
 
-If you're not familiar with continuous integration and don't already have a repository that you want to build, this guide will show you what it is all about.
+Welcome to Travis CI! If you are new here or new to Continuous Integration (CI) in general, this doc will provide context and terminology used throughout the platform and documentation. 
 
-You need to sign up for a [GitHub account](https://github.com/) if you do not already have one.
+### What is Continuous Integration (CI)?
 
-### To get started with Travis CI:
+Continuous Integration is the concept and practice of merging in small code changes frequently - rather than waiting for a large integration process at the end of a development cycle. Companies and individuals may all do this process differently, but the goal is to build healthier software by developing and testing in smaller increments. This is where Travis CI can be helpful.
 
-1. On GitHub, fork the [example PHP repository](https://github.com/plaindocs/travis-broken-example).
+Travis CI, as a CI platform, supports your development process by automatically building and testing code changes, in order to provide immediate feedback on the success of the change. Travis CI also can automate some of your development process by managing deployments and notifications.  
 
-2. [Sign in to Travis CI](https://travis-ci.org/auth) with your GitHub account, accepting the GitHub [access permissions confirmation](/user/github-oauth-scopes).
+### CI builds and automation: building, testing, deploying
 
-3. Once you're signed in, and we've synchronized your repositories from GitHub, go to your [profile page](https://travis-ci.org/profile) and enable
-   Travis CI builds for your fork of the `travis-broken-example` repository.
+Many things can happen when Travis CI runs. First, your project is brought into into our clean environment with `git clone`, then your code is compiled. Next, any tests you have are run, and their status is checked. If tests are successful, and you have set up a deployment provider, Travis CI will deploy your application. Depending on your settings, you may be notified via email, or over other platforms such as Slack. 
 
-   > Note: You can only enable Travis CI builds for repositories you have admin access to.
+One of the advantages of CI builds, is they can execute arbitrary code if you configure the build to run a custom script. This means you can setup your own notifications, upload files output during your build (artifacts), and many other processes. Travis CI can function as a basis for general automation for your development processes. 
 
-4. Take a look at `.travis.yml`, the file which tells Travis CI what to do:
+You can also customize your build into various stages, in which later stages are dependent on the result of prior stages. See our [Build Stages](../build-stages) document for details. 
```

Comment:
@plaindocs We also might have discussed this as well, but I think we should include reference to it here as part of "things you can automate", or drop it entirely? Thoughts?